### PR TITLE
Add SendViewBCH / SendFormBCH

### LINKED ISSUE #994: [BCH] Add Send component
As part of #821

### PR DESCRIPTION
**Pull request by veado**
- [x] Add `SendViewBCH` + `SendFormBCH`
- [x] Add stories for `SendFormBCH` + `SendFormLTC`

Closes #994

### CODE REVIEW

File: src/renderer/components/wallet/txs/send/SendFormBCH.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  formatBaseAmount
+} from '@xchainjs/xchain-util'
+
+import { mockValidatePassword$ } from '../../../../../shared/mock/wallet'
+import { BCH_DECIMAL } from '../../../../helpers/assetHelper'
+import { SendTxParams } from '../../../../services/chain/types'
+import { WalletBalance } from '../../../../types/wallet'
+import { SendFormBCH as Component, Props as ComponentProps } from './SendFormBCH'
+
+const btcBalance: WalletBalance = {
```
Comment (junkai121) [inline src/renderer/components/wallet/txs/send/SendFormBCH.stories.tsx:22]: `btcBalance` should be `bchBalance`?

File: src/renderer/components/wallet/txs/send/SendFormLTC.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  formatBaseAmount
+} from '@xchainjs/xchain-util'
+
+import { mockValidatePassword$ } from '../../../../../shared/mock/wallet'
+import { LTC_DECIMAL } from '../../../../helpers/assetHelper'
+import { SendTxParams } from '../../../../services/chain/types'
+import { WalletBalance } from '../../../../types/wallet'
+import { SendFormLTC as Component, Props as ComponentProps } from './SendFormLTC'
+
+const btcBalance: WalletBalance = {
```
Comment (junkai121) [inline src/renderer/components/wallet/txs/send/SendFormLTC.stories.tsx:22]: Same here, btcBalance -> ltcBalance?

**Review by junkai121 — CHANGES_REQUESTED** [review]
LGTM! just small nitpick for stories...

File: src/renderer/components/wallet/txs/send/SendFormBCH.tsx [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+
+  const renderFeeError = useMemo(() => {
+    if (!isFeeError) return <></>
+
+    const msg = intl.formatMessage(
+      { id: 'wallet.errors.fee.notCovered' },
+      {
+        balance: formatAssetAmountCurrency({
+          amount: baseToAsset(balance.amount),
+          asset: AssetBTC,
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/wallet/txs/send/SendFormBCH.tsx:156]: is it correct to use BTC asset here ?

File: src/renderer/components/wallet/txs/send/SendFormBCH.tsx [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+        <></>
+      ),
+    [sendHandler, showPwModal, validatePassword$]
+  )
+
+  const uiFeesRD: UIFeesRD = useMemo(
+    () =>
+      FP.pipe(
+        feeRD,
+        RD.map((fee) => [{ asset: AssetBTC, amount: fee }])
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/wallet/txs/send/SendFormBCH.tsx:293]: same question about BTC asset

**Review by thatStrangeGuyThorchain — APPROVED** [review]
just few small questions

File: src/renderer/components/wallet/txs/send/SendFormBCH.stories.tsx (hunk 1 above)
Comment (veado) [reply src/renderer/components/wallet/txs/send/SendFormBCH.stories.tsx:22]: Good catch! _Going to fix it..._

File: src/renderer/components/wallet/txs/send/SendFormLTC.stories.tsx (hunk 2 above)
Comment (veado) [reply src/renderer/components/wallet/txs/send/SendFormLTC.stories.tsx:22]: Yep, same here :) - good catch!

File: src/renderer/components/wallet/txs/send/SendFormBCH.tsx (hunk 3 above)
Comment (veado) [reply src/renderer/components/wallet/txs/send/SendFormBCH.tsx:156]: Oh no, it should be `AssetBCH` of course - another good catch!

File: src/renderer/components/wallet/txs/send/SendFormBCH.tsx (hunk 4 above)
Comment (veado) [reply src/renderer/components/wallet/txs/send/SendFormBCH.tsx:293]: Yep, BCH should be there, thx for pointing it out!

**Pull request merged**